src/command-dispatcher.ts

```diff
@@ -0,0 +1,82 @@
+import {  } from "./server";
```

Comment:
Why do you have an empty import first, and then a `require`? 
Would it be possible to use only `import` instead of `require`?